dev/com.ibm.ws.security.jwt/resources/com/ibm/ws/security/jwt/internal/resources/JWTMessages.nlsprops

```diff
@@ -277,9 +277,9 @@ JWT_SIGNER_CERT_NOT_AVAILABLE.useraction=Verify that the truststore that is used
 # Do not translate "JWT", "trustedAlias"
 # single quoted terms do NOT use double single quotes when there are no replacement variables
 # Intended to be a child of JWT_ERROR_GETTING_PUBLIC_KEY
-JWT_SIGNER_CERT_AMBIGUOUS=CWWKS6047E: Cannot retrieve a signing key from the truststore. The 'trustedAlias' attribute was not specified in the JWT consumer configuration and there are multiple signer certificates in the truststore.
-JWT_SIGNER_CERT_AMBIGUOUS.explanation=Multiple signer certificates were found in the configured truststore. When there are multiple certificates in the truststore, the 'trustedAlias' attribute must be specified in the JWT consumer configuration to establish which of the certificates to use.
-JWT_SIGNER_CERT_AMBIGUOUS.useraction=Specify the 'trustedAlias' JWT consumer configuration attribute with the alias of the appropriate signer certificate.
+JWT_SIGNER_CERT_AMBIGUOUS=CWWKS6047E: Cannot determine which signer certificate in the truststore to use. Add the 'trustedAlias' attribute to the JWT consumer configuration or the 'keyName' attribute to the MP-JWT configuration.
+JWT_SIGNER_CERT_AMBIGUOUS.explanation=Multiple signer certificates were found in the configured truststore. When there are multiple certificates in the truststore, the 'trustedAlias' attribute must be specified in the JWT consumer configuration or the 'keyName' attribute must be specified in the MP-JWT configuration to establish which of the certificates to use.
+JWT_SIGNER_CERT_AMBIGUOUS.useraction=Specify the 'trustedAlias' JWT consumer configuration attribute or the 'keyName' MP-JWT configuration attribute with the alias of the appropriate signer certificate.
```

Comment:
I was a little confused here but I think I understand. To fix the ambiguity, the user needs to specify which certificate in the truststore should be used. Will adding the attribute to either 'kevNAME' or 'trustedAlias' produce the same effect? I've rephrased the message a little to make the relationship a little clearer that you can fix the ambiguity by specifying the certificate to use and that you can specify the certificate by changing one of two configuration attributes. 

How does this look to you?  
"Specify which certificate to use in the truststore by adding the alias of the appropriate signer certificate to either the 'trustedAlias' JWT consumer configuration attribute or the 'kevNAME' MP-JWT configuration attribute."

Reply:
both changes have been added, thanks.

#build